william000000/Banka-
# Review API Endpoint and Make its Test

## What does this PR do?

- Recollect API, Fix all bugs about testing and set up Travis

## Description of the task to be completed
- Test my Endpoint
- Review my code and make my endpoint accurate
- Setup travis badge
- New Validation

## What are relevant pivotal tracker stories
## How can it be manually tested
- clone my repo then after checkout to this branch `dev` 
- Install all dependencies by this command `npm install`
- After all the above steps, you can check any route and then check using postman for the result
